src/libcore/num/wrapping.rs

```diff
@@ -119,6 +119,16 @@ macro_rules! wrapping_impl {
             }
         }
 
+        #[stable(feature = "rust1", since = "1.3.0")]
+        impl Div for Wrapping<$t> {
+            type Output = Wrapping<$t>;
+
+            #[inline(always)]
+            fn div(self, other: Wrapping<$t>) -> Wrapping<$t> {
+                Wrapping(self.0 / other.0)
```

Comment:
This should look like `Wrapping(self.0.wrapping_div(other.0))`; currently your code will panic rather than overflow on MIN/-1 ∀ signed integers.